bitnami/spark/templates/configmap.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "spark.fullname" . }}-config
```

Comment:
I checked and the configmap is not being used at all

```
 1 ❯ kubectl get pods -o yaml  | grep opining-whale-spark-config
 ~                                                                            ```

Reply:
You are right, that is an error I made after change some things, I have fixed it.